articles/quickstart/webapp/aspnet-core/02-user-profile.md

```diff
@@ -84,19 +82,12 @@ public void ConfigureServices(IServiceCollection services)
             NameClaimType = "name"
         };
 
-        options.Events = new OpenIdConnectEvents
-        {
-            // handle the logout redirection 
-            OnRedirectToIdentityProviderForSignOut = (context) =>
-            {
-                //...
-            }
-        };
+        //...
     });
 }
 ```
 
-Next create a view. For the view, display user's profile card at the top with the user's name, email address and profile image.
+Next create a view called Profile.cshtml inside `Views/Account` and display the user's name, email address and profile image.
```

Comment:
I'd add backticks around filenames, too.

```suggestion
Next create a view called `Profile.cshtml` inside `Views/Account` and display the user's name, email address and profile image.
```

Reply:
I made the exact change in the ASPNET Core 3.1 quickstart as part of this PR.